src/jupyter_contrib_nbextensions/nbextensions/toc2/toc2.js

```diff
@@ -176,15 +177,15 @@ var make_link = function(h, num_lbl) {
                 $('#notebook-container').css('margin-left', 30);
                 $('#notebook-container').css('width', $('#notebook').width() - 30);
             } else { // original width
-              $("#notebook-container").css({'width':"82%", 'margin-left':'auto'})             
+              $("#notebook-container").css({'width':st.nbcontainer_width, 'margin-left':'auto'})             
```

Comment:
I think this should probably set an empty string, to not set anything, otherwise it'll break when the window is resized.